docs/how-to-run-example-android.md

```diff
@@ -23,44 +23,24 @@ $ export PATH=$PATH:${PWD}/android-ndk-r21d/
 $ export ANDROID_NDK=${PWD}/android-ndk-r21d/
 ```
 
-### Build NNTrainer with NDK-Build
+## Build NNTrainer with NDK-Build
 Once you install NDK package, you can build the nntrainer.so for android as below.
 Currently, the APP_ABI is set arm64-v8a. If you want to use armeabi-v7a, you have to change in Application.mk file.
 
 ```bash
 $ ls
-api                 CONTRIBUTING.md  index.md  MAINTAINERS.md     nnstreamer       packaging
-Applications        debian           jni       meson.build        nntrainer        README.md
-CODE_OF_CONDUCT.md  docs             LICENSE   meson_options.txt  nntrainer.pc.in  test
+api                 CONTRIBUTING.md  index.md  MAINTAINERS.md     nnstreamer        nntrainer.pc.in  RELEASE.md
+Applications        debian           jni       meson.build        nntrainer         packaging        test
+CODE_OF_CONDUCT.md  docs             LICENSE   meson_options.txt  nntrainer.ini.in  README.md        tools
 $
-$ export NNTRAINER_ROOT=${PWD}
-$
-$ cd ./jni
-Android.mk      prepare_iniparser.sh  prepare_tflite.sh
-Application.mk  prepare_openblas.sh
-$ ndk-build NDK_PROJECT_PATH=./ APP_BUILD_SCRIPT=./Android.mk NDK_APPLICATION_MK=./Application.mk -j $(nproc)
-Android.mk:19: INIPARSER_ROOT is not defined!
-Android.mk:20: INIPARSER SRC is going to be downloaded!
-Cloning into 'iniparser'...
-~/nntrainer_new/jni ~/nntrainer_new/jni PREPARING ini parser at . ~/nntrainer_new/jni
-Android.mk:35: BUILDING TFLITE BACKBONE !
-Android.mk:40: TENSORFLOW_ROOT is not defined!
-Android.mk:41: TENSORFLOW SRC is going to be downloaded!
-PREPARING TENSORFLOW 1.13.1 at . ~/nntrainer_new/jni ~/nntrainer_new/jni [TENSORFLOW-LITE] Download tensorflow-1.13.1 [TENSORFLOW-LITE] Finish downloading tensorflow-1.13.1 [TENSORFLOW-LITE] untar tensorflow-1.13.1 ~/nntrainer_new/jni
-[arm64-v8a] Compile++      : nntrainer <= tflite_layer.cpp
-In file included from ././../nntrainer/layers/tflite_layer.cpp:15:
-In file included from ./../nntrainer/layers/tflite_layer.h:18:
-In file included from ./../nntrainer/layers/layer_internal.h:31:
-...
-$ ls
-Android.mk      iniparser  obj                   prepare_openblas.sh  tensorflow-1.13.1
-Application.mk  libs       prepare_iniparser.sh  prepare_tflite.sh
-$ ls libs/arm64-v8a
-libcapi-nntrainer.so  libccapi-nntrainer.so  libc++_shared.so  libnntrainer.so
-$ mv libs ../
+$ ./tools/package_android.sh
+$ ls builddir/android_build_result
+Android.mk  conf  examples  include  lib
+$ ls builddir/android_build_result/libs/arm64-v8a
+libcapi-nntrainer.so  libccapi-nntrainer.so  libc++_shared.so  libnnstreamer-native.so  libnntrainer.so
```

Comment:
May be add another way to build as usual as below.

```
meson builddir -Dbuildtype=android  ( other options )
cd builddir
ninja
ninja install
```

Reply:
Fixed